cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -55,27 +56,32 @@ import (
 
 // CMServer is the main context object for the controller manager.
 type CMServer struct {
-	Port                    int
-	Address                 util.IP
-	CloudProvider           string
-	CloudConfigFile         string
-	ConcurrentEndpointSyncs int
-	ConcurrentRCSyncs       int
-	ServiceSyncPeriod       time.Duration
-	NodeSyncPeriod          time.Duration
-	ResourceQuotaSyncPeriod time.Duration
-	NamespaceSyncPeriod     time.Duration
-	PVClaimBinderSyncPeriod time.Duration
-	RegisterRetryCount      int
-	NodeMonitorGracePeriod  time.Duration
-	NodeStartupGracePeriod  time.Duration
-	NodeMonitorPeriod       time.Duration
-	NodeStatusUpdateRetry   int
-	PodEvictionTimeout      time.Duration
-	DeletingPodsQps         float32
-	DeletingPodsBurst       int
-	ServiceAccountKeyFile   string
-	RootCAFile              string
+	Port                             int
+	Address                          util.IP
+	CloudProvider                    string
+	CloudConfigFile                  string
+	ConcurrentEndpointSyncs          int
+	ConcurrentRCSyncs                int
+	ServiceSyncPeriod                time.Duration
+	NodeSyncPeriod                   time.Duration
+	ResourceQuotaSyncPeriod          time.Duration
+	NamespaceSyncPeriod              time.Duration
+	PVClaimBinderSyncPeriod          time.Duration
+	PVDefaultScrubPodPath            string
+	NFSScrubPodMinimumTimeout        int
+	NFSScrubPodTimeoutIncrement      int
+	HostPathScrubPodMinimumTimeout   int
+	HostPathScrubPodTimeoutIncrement int
```

Comment:
@pmorie I'm thinking about your config/schema idea.   Maintaining versions might be too much for this PR, but I wonder if making a single, serializable struct for config makes for a cleaner CLI?   It would be easier to specify a file path to a volumeConfig than to specify all these different flags on the CLI.

Thoughts?